taxonomies/categories.txt

```diff
@@ -86244,7 +86300,7 @@ en:Powdered sugars, Confectioner's sugars, Icing sugars
 de:Puderzucker, Staubzucker, Dekorierzucker
 es:Azúcar glas, Azúcar en polvo, Azúcar glacé
 fi:tomusokerit
-fr:Sucres glaces, Sucres impalpables, Sucres à glacer, Sucre glace
+fr:Sucres glaces, Sucres impalpables, Sucres à glacer, Sucre glace, Sucres en poudre, Sucre en poudre
```

Comment:
not quite the same thing

Reply:
https://en.wikipedia.org/wiki/Powdered_sugar

Reply:
From the wikipedia: "Powdered Sugar, also called confectioners' sugar, 10X sugar or icing sugar".

Or are these different in french?

Reply:
@aleene It is different in French, "sucre en poudre" is "caster sugar" : a bit finer than "granulated sugar", but much much coarser than icing sugar ("sucre glace").

See http://www.cultures-sucre.com/autour-du-sucre/a-chaque-sucre-son-usage/

(in Canada, "powdered sugar" is translated litterally as "sucre en poudre").

Reply:
OK reverted